tests/test_file_transfer_service.py

```diff
@@ -52,3 +56,39 @@ def test_remote_grpc_fts_container(monkeypatch, new_solver_session, new_mesh_ses
         )
         assert meshing.file_exists_on_remote("downloaded_meshing_mixing_elbow.msh.h5")
         assert file_downloaded_to_the_client("downloaded_meshing_mixing_elbow.msh.h5")
+
+
+@pytest.mark.skip(
+    reason="Unable to copy data file to Fluent's current working directory."
+)
+@pytest.mark.standalone
+def test_read_case_and_data():
+    import ansys.fluent.core as pyfluent
+
+    case_file_name = examples.download_file(
+        "mixing_elbow.cas.h5", "pyfluent/mixing_elbow"
+    )
+    data_file_name = examples.download_file(
+        "mixing_elbow.dat.h5", "pyfluent/mixing_elbow"
+    )
+    assert case_file_name
+    assert data_file_name
+    solver = pyfluent.launch_fluent(file_transfer_service=LocalFileTransferStrategy())
+
+    with pytest.raises(FileNotFoundError):
```

Comment:
This line should not be there if the test is already skipped. When we fix the code, we shall just remove the skip marker.

Reply:
Add a code comment instead above the line that fails to indicate it, so that we still have this documented. But without the `pytest.raises()`

Reply:
Done. Thank you.